frontend/src/components/AdequacyDoughnut/AdequacyDoughnut.tsx

```diff
@@ -24,6 +26,16 @@ type Props = {
 
 export let AdequacyDoughnut = withStore('adequacies', 'method')<Props>(({ serviceAreas, store }) => {
 
+  if (isEmpty(store.get('adequacies'))) {
+    return <div className='AdequacyDoughnut Flex -Center'>
+      <CircularProgress
+        size={150}
+        thickness={8}
+        color={'#3f51b5'}
```

Comment:
@bcherny Material-UI doesn't expose a className attribute for Circular Progress for some reason. Is there another way to override the css to set this color using a variable? or is there a way to pull the css variable into this file and use it?